crates/bitwarden/src/secrets_manager/secrets/get.rs

```diff
@@ -18,7 +18,6 @@ pub(crate) async fn get_secret(
 ) -> Result<SecretResponse> {
     let config = client.get_api_configurations().await;
     let res = bitwarden_api_api::apis::secrets_api::secrets_id_get(&config.api, input.id).await?;
-
     let enc = client.get_encryption_settings()?;
```

Comment:
```suggestion

    let enc = client.get_encryption_settings()?;
```

Reply:
Resolved in [429a1ff](https://github.com/bitwarden/sdk/pull/388/commits/429a1ffd54d33d5c37d9124e121e829422618869)